src/middlewares/createContextMiddleware.ts

```diff
@@ -65,13 +100,19 @@ export const createContextMiddleware = ({
 
     ctx.setMyCommands(CommandDescriptions);
 
-    if (!interval) {
-      interval = setInterval(() => callMessageRecycling(), 5000);
-    }
+    const chatMessageRecyclingInterval =
+      messageDeletionIntervals[chatId];
 
-    const callMessageRecycling = () => {
-      runMessageRecycling(ctx);
-    };
+    if (!chatMessageRecyclingInterval) {
+      ctx.logger.info('Creating auto delete interval.');
```

Comment:
This message should also be more descriptive, at least tell for which chat id is this interval being created.

Reply:
fixed